app/component/itinerary/navigator/navigator.scss

```diff
@@ -601,6 +612,16 @@
   }
 }
 
+@keyframes hideTopCard {
+  from {
+    transform: translateY(0);
+  }
+
+  to {
+    transform: translateY(-100%);
+  }
+}
+
```

Comment:
Probably same as `@keyframes slideUpToTop`

Reply:
SlideUpToTop has this property
`
  to {
    transform: translateY(calc(-100vh - 100%));
  }
}`

And it did not respond well with this animation. I should double check that though!

Reply:
The animation which moves navi-card-container completely outside viewport uses `slideUpToTop`. If I interpret the desired outcome of `hideTopCard` animation correctly, the two _should_ use identical keyframes. However, the mileage may vary as `slideUpToTop` aims to move both the top card and any extensions or extra cards below it outside viewport.

Reply:
Double checked, and some reason that proprety messes up the animation. So I suggest we keep my change as is.

Reply:
Let's keep yours and then refactor later if possible